src/parser.ts

```diff
@@ -696,6 +760,10 @@ function validateArity(name: string, arity: GroqFunctionArity, count: number) {
   }
 }
 
+function argumentShouldBeSelector(namespace: string, args: NodeTypes.ExprNode[]) {
+  return namespace == 'diff' && args.length == 2
```

Comment:
I think this needs to be more specific about the function names it tests for. Also, `diff::changedAny()` and `diff::changedOnly()` take three arguments, not two, and only the last argument is a selector.